docs/cloud-storage-fileset-example.md

```diff
@@ -0,0 +1,676 @@
+---
+title: "How to use cloud storage fileset"
+slug: /how-to-use-cloud-storage-fileset
+keyword: fileset S3 GCS ADLS OSS
+license: "This software is licensed under the Apache License version 2."
+---
+
+This document aims to provide a comprehensive guide on how to use cloud storage fileset created by Gravitino, it usually contains the following sections:
+
+
+## Start up Gravitino server
+
+### Start up Gravitino server
+
+Before running the Gravitino server, you need to put the following jars into the fileset class path located in `${GRAVITINO_HOME}/catalogs/hadoop/libs`. For example, if you are using S3, you need to put gravitino-aws-hadoop-bundles-{version}.jar into the fileset class path.
```

Comment:
Also here, what is "fileset classpath"? Also, be aware that "classpath" is a word, not "class path".

Reply:
Besides, use "``" to quote this "gravitino-aws-hadoop-bundles-{version}.jar", otherwise the docusaurus will be failed.

Reply:
> Besides, use "``" to quote this "gravitino-aws-hadoop-bundles-{version}.jar", otherwise the docusaurus will be failed.

It's safe to use it as there is no `$` here.